posts/taking-learning-outside-the-classroom-into-the-cloud.md

```diff
@@ -0,0 +1,54 @@
+
+My first engagement with this team involved upgrading our rule testing platform to a higher tier of resiliency, by making the infrastructure “active/active.” Despite the intimidation of my first assignment, I was still excited to put my abilities to learn on-the-fly to the test. I paired with a teammate who joined the company at the same time I did, and together we learned about Jenkins pipelines, AWS Lambda/ELB/EC2/Route53 and their interactions with one another, as well as how to deploy them all using the company's custom deployment processes. These technologies were just the tip of the iceberg-sized tech stack that I worked with over the next nine months: among others, I got hands-on experience with Docker, Kubernetes, and Maven (maybe even a little too much experience with Maven dependency management... if you know, you know).
+
+More recently, I was able to wipe the dust off my skills designing user experiences while redesigning components for our rules testing platform. As the company matures, a common goal among leaders has been the idea of convergence. As such, other organizations have begun looking to this rules testing platform to accelerate and streamline what tends to be a clunky and time-consuming validation process. Whereas during my internship, I was only designing for one user, I quickly realized how much more of a balancing act it is to design for multiple use cases – some of which we don't necessarily realize exist yet. Many of the questions we asked led to dozens more questions asked than we started with, and some of the assumptions we had made ended up flipped on their heads. But what helped guide us through every complicated discussion and change in plans was calling out risks early on, readiness to adapt, and sticking to a concrete set of design principles.
+
+If I had to pick the most surprising component of my work with this client, however, it would be the few times I went "on call" for 24 hours at a time as the first line of defense to answer any alerts triggered by issues in production. While I definitely won't miss being woken up in the middle of the night by mysterious robo-calls, there was a certain level of importance and impact that I felt in being that first line of support. Type 2 fun, let's call it.
```

Comment:
```suggestion
If I had to pick the most surprising component of my work with this client, it would be the few times I went "on call" for 24 hours at a time as the first line of defense to answer any alerts triggered by issues in production. While I won't miss being woken up in the middle of the night by mysterious robo-calls, there was a certain level of importance and impact that I felt in being that first line of support. Type 2 fun, let's call it.
```